[2024-07-15T08:16:38.824+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-15T08:16:38.836+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: user_processing.create_table manual__2024-07-15T08:16:36.787131+00:00 [queued]>
[2024-07-15T08:16:38.860+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: user_processing.create_table manual__2024-07-15T08:16:36.787131+00:00 [queued]>
[2024-07-15T08:16:38.862+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-15T08:16:38.873+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): create_table> on 2024-07-15 08:16:36.787131+00:00
[2024-07-15T08:16:38.879+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=725) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-15T08:16:38.879+0000] {standard_task_runner.py:63} INFO - Started process 732 to run task
[2024-07-15T08:16:38.881+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2024-07-15T08:16:36.787131+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpa7snr4mx']
[2024-07-15T08:16:38.882+0000] {standard_task_runner.py:91} INFO - Job 33: Subtask create_table
[2024-07-15T08:16:38.911+0000] {task_command.py:426} INFO - Running <TaskInstance: user_processing.create_table manual__2024-07-15T08:16:36.787131+00:00 [running]> on host 6e5e891f9a78
[2024-07-15T08:16:38.957+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='user_processing' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2024-07-15T08:16:36.787131+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-15T08:16:36.787131+00:00'
[2024-07-15T08:16:38.958+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-15T08:16:38.958+0000] {sql.py:276} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        
[2024-07-15T08:16:38.962+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-07-15T08:16:38.966+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2024-07-15T08:16:38.968+0000] {sql.py:487} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2024-07-15T08:16:38.969+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-15T08:16:38.973+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, run_id=manual__2024-07-15T08:16:36.787131+00:00, execution_date=20240715T081636, start_date=20240715T081638, end_date=20240715T081638
[2024-07-15T08:16:39.013+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-15T08:16:39.022+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-15T08:16:39.023+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
